Text Formatting.html

```diff
@@ -242,41 +242,42 @@ <h3><a name="Syntax">Format string syntax</a></h3>
 <li>There are subtle differences between different implementations. For example,
     POSIX positional arguments <a href="#2">[2]</a> are not supported on
     some systems <a href="#6">[6]</a>.</li>
-<li>Using <code>'%'</code> in a custom format specifier, e.g. for
-    <code>put_time</code>-like time formatting, poses difficulties.</li>
+<li>Using <code>'%'</code> in a custom format specifier poses difficulties, e.g.
+for <code>put_time</code>-like time formatting.</li>
 </ul>
 
 <p>
 Although it is possible to address these issues while maintaining resemblance
-to the original printf format, this will still break compatibility and can
-potentially be more confusing to users than introducing a different syntax.
+to the original <code>printf</code> format, this will still break compatibility
+and can potentially be more confusing to users than introducing a different
+syntax.
 </p>
 
 <p>
 Therefore we propose a new syntax based on the ones used in Python
 <a href="#3">[3]</a>, the .NET family of languages <a href="#4">[4]</a>,
 and Rust <a href="#5">[5]</a>. This syntax employs <code>'{'</code> and
 <code>'}'</code> as replacement field delimiters instead of <code>'%'</code>
-and it is described in details in the <a href="#SyntaxRef">syntax reference</a>.
-Here are some of the advantages:
+and it is described in detail in the <a href="#SyntaxRef">syntax reference</a>.
+Some advantages of the proposal are:
 </p>
 
 <ul>
-<li>Consistent and easy to parse mini-language focused on formatting rather
+<li>A consistent and easy to parse mini-language focused on formatting rather
     than conveying type information</li>
 <li>Extensibility and support for custom format strings for user-defined
     types</li>
 <li>Positional arguments</li>
 <li>Support for both locale-specific and locale-independent formatting (see
     <a href="#Locale">Locale support</a>)</li>
-<li>Formatting improvements such as better alignment control, fill character,
-    and binary format
+<li>Formatting improvements such as better alignment control, character filling,
```

Comment:
I'd keep "fill character" because it's an established term for this part of the format string syntax and used elsewhere in the paper.

Reply:
Sure, that makes sense.